prihlas-me.php

```diff
@@ -0,0 +1,38 @@
+<?php
+require_once "sdilene.php";
+
+$uzivatelskeJmeno = '';
+$uzivatelskeHeslo = '';
+
+// pokud neexistuje session, zalozime si ji
+if (!isset($_SESSION['pokusu_prihlaseni'])) {
+    $_SESSION['pokusu_prihlaseni'] = 0;
+}
+
+// pokud se opravdu uzivatel snazi prihlasit, zapocitame mu to a zobrazime hlasku
+if (isset($_POST['jmeno']) && isset($_POST['heslo'])) {
+    $_SESSION['pokusu_prihlaseni']++;
+
+    $_SESSION["status-prihlaseni"] = 403;
```

Comment:
Tímhle si stále nejsem jistý. 

Pokud to tady necháš, tak bych jim alespoň v rychlosti vysvětlil co je to http kód a k čemu se používá.

Reply:
Já bych tam kódy netahal. Nemá to imho kontext, a těžko chytnou, proč to vlastně dělat. Kduž už, zkus jít cestou těch flash mesagess, kde je tam i text (kde je napsaný, co se stalo), a dá se vysvětlit a nakreslit a na postitech ukázat, že tam přidáš zprávu, příště jí zobrazíš a array_pop ji dáš pryč ať ji nezobrazuješ zase příště a tradá.